queries.sql

```diff
@@ -111,4 +111,42 @@ GROUP BY  species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts
 FROM animals
 WHERE  date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+SELECT o.full_name, a.name
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+
+SELECT s.name, COUNT(*) as animal_count
+FROM animals a
+JOIN species s ON a.species_id = s.id
+GROUP BY s.name;
```

Comment:
- [ ] Again here kindly fix your queries and data so as to get the desired result for the query to list `How many animals are there per species`

Your output 
![image](https://github.com/waelelsafty07/vet/assets/81932478/2767895e-4aa5-4d2c-b4b6-0c77363b3f54)

Expected Output
![image](https://github.com/waelelsafty07/vet/assets/81932478/d388d98a-b7af-4f8e-8ea2-00ec2e817e00)

#### ScreenShot will have to be updated